tests/api/commerce_decoupled_checkout/GiftCest.php

```diff
@@ -0,0 +1,212 @@
+<?php
+
+namespace falcon_commerce;
+
+use Codeception\Util\HttpCode;
+use Drupal\commerce_price\Price;
+
+/**
+ * Class GiftCest.
+ *
+ * Gift API tests.
+ *
+ * @package Falcon Commerce
+ */
+class GiftCest {
+
+  private $variation;
+
+  private $product;
+
+  /**
+   * @var array
+   */
+  private $post = [
+    'order' => [
+      'type' => 'gift',
+      'order_items' => [
+        [
+          'type' => 'gift',
+          'unit_price' => [
+            'number' => 15,
```

Comment:
Why does it have this price?

Reply:
Today i looked to commerce_decoupled_checkout module and i think, the unit_price don`t need in this case.